optaplanner-core/src/main/java/org/optaplanner/core/impl/score/stream/drools/common/BiTuple.java

```diff
@@ -18,15 +18,15 @@
 
 import java.util.Objects;
 
-public final class BiTuple<K,V> {
-    public final K key;
-    public final V value;
+public final class BiTuple<A, B> {
+    public final A _1;
+    public final B _2;
     private final int hashCode;
 
-    public BiTuple(K key, V value) {
-        this.key = key;
-        this.value = value;
-        this.hashCode = Objects.hash(key, value);
+    public BiTuple(A _1, B _2) {
```

Comment:
Why not use variable names "a" and "b" instead of "_1" and "_2"?

Reply:
It's all the same to me. I can change it.